docs/pages/modules/module-api.md

```diff
@@ -1,32 +1,29 @@
 ---
-title: Module API
+title: Native Modules
 ---
 
 import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
-import { PlatformTag } from '~/components/plugins/PlatformTag';
 import { APIBox } from '~/components/plugins/APIBox';
 
-> Note: This API is still experimental and subject to change. Some features that you need may not be implemented yet.
-
-The Expo module API provided by `expo-modules-core` is an abstraction layer on top of React Native modules that helps you build native modules in modern languages (Swift and Kotlin) with an easy to use and convenient API that is consistent across platforms where possible.
+The Expo Modules API for native modules is an abstraction layer on top of React Native modules that helps you build native modules in modern languages (Swift and Kotlin) with an easy to use and convenient API that is consistent across platforms where possible.
```

Comment:
i still think "Expo Modules API" sounds wrong to me but let's chat about your reasoning here on our sync :)

Reply:
@ide testing out unblocked here ;)

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/5bf29d2c-23ae-4af8-8ccc-fe767cd2df79/thread/1a1dcec2-e22b-4515-bbe2-625436bdae9d?message=f21f71c2-119c-4d66-93bb-e7a5f998a89a).

Reply:
So what about 👇?
```suggestion
The Expo native modules API is an abstraction layer on top of React Native modules that helps you build native modules in modern languages (Swift and Kotlin) with an easy to use and convenient API that is consistent across platforms where possible.
```

Reply:
works for me!